packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss

```diff
@@ -17,7 +17,7 @@
   }
 
   .img {
-    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    color: var(--terra-consumer-logo-img-color, #3d3d3d);
```

Comment:
this color is applied on alternate text. If the src-url is valid then logo image will be rendered or else alt text will be rendered in this color.

Reply:
we can use - `--terra-consumer-nav-text-color` here. Also mentioned in [jira description](https://jira2.cerner.com/browse/PORTALDEV-36272)

Reply:
-logo-text-color and -logo-img-color are different so I'm reverting it back to as it was earlier.